django/webscraper/web_scraper.py

```diff
@@ -117,6 +138,72 @@ def get_links_to_all_animal(href: str) -> set[str]:
     return result
 
 
+def scrape():
+    """Does full webscrapping and adds new data to database.
+    Writes webscrapper progress in webscrapper_log.txt
+    Note: DO NOT use outside start_webscrapping()
+
+    Usage example
+    scrape()
+    ...
+    """
+    pet_links = get_links_to_all_animal(
+        "https://napaluchu.waw.pl/zwierzeta/znalazly-dom"
+    )
+    pet_iter = 1
+    for pet_link in pet_links:
+        pet_info = parse_pet(pet_link)
+        create_pet(pet_info=pet_info, href=pet_link)
+        logging.info("Finished pet: %lu\n", pet_iter)
+        pet_iter += 1
+    clear_webscrapping_processes()
+
+
+def kill_scrapping():
+    """Kills process currently scrapping if there is one - else
+    does nothing.
+
+    Usage example
+    kill_scrapping()
+    ...
+    """
+    try:
+        proc = WebscrappingProcess.objects.get(working=True)
+    except WebscrappingProcess.DoesNotExist:
+        clear_webscrapping_processes()
+        return
+    pid = proc.pid
+    proc.delete()
+    os.kill(pid, signal.SIGKILL)
+
+    logging.info("Webscrapping stopped\n")
+
+
+def start_scrapping():
+    """Starts (or restarts) the scrapping process

```

Comment:
```suggestion
    """Starts the scrapping process. In case there another process running, it gets terminated."""

```

Reply:
It would seem to me that my comment is more readable - perhaps because of existence of a verb.

Reply:
I'm sorry for the typo.

Reply:
I think this should be changed due to the uncertainties about the function. Does it start or restart the process. When does it start or restarts. It's unclear.